### ISSUE TITLE
Klassisk elektromagnetisme

### ISSUE THREAD

**Issue by danmichaelo** [realfagstermer/realfagstermer#235]
`Klassisk elektromagnetisme` og `Elektromagnetisme` kan vel slås sammen. Er ikke `Elektromagnetisme` "per definisjon" klassisk @mittinatten ?

**Comment by mittinatten** [comment]
Jeg ser ingen grunn til å skille på de, nei

**Comment by mittinatten** [comment]
Vi har også `Elektrodynamikk`. Jeg har sett lite i oppslagsverkene og finner inget klart skille mot `Elektromagnetisme`.  Muligens at elektromagnetisme brukes om fenomenet og elektrodynamikk om teorien. Wikipedia samler de under en artikkel. Kanskje lurt å beholde `Klassisk ...` som foretrukket for å unngå forvirring med Kvanteelektrodynamikk.

     Klassisk elektrodynamikk
     BF Elektromagnetisme
     BF Elektrodynamikk
     BF Klassisk elektromagnetisme
     SO Kvanteelektrodynamikk

Med problemet at det ser ut som at en del bøker om QED har fått emnet `Elektrodynamikk`. Hvis vi bruker `Elektrodynamikk` som foretrukket forsvinner problemet (eller det skjules bedre :) ).

**Comment by danmichaelo** [comment]
Ja, søren, jeg husker jeg også begynte å tenke på `Elektrodynamikk`, og så grublet jeg meg fast uten å få skrevet noe :grimacing: Men jeg synes forslaget er bra! Helt enig i at de bør slås sammen. Veldig usikker på hva som bør være foretrukken term, men `Klassisk elektrodynamikk` er ihvertfall ingen dårlig kandidat, så vi kan godt gå for den.

**Comment by danmichaelo** [comment]
Tok forresten en liten rydderunde og omklassifiserte bøker som helt tydelig handlet om QED. Så ja, det kan nok være et poeng å ha med "Klassisk" for å hindre sammenblanding :)

**Comment by bibliomari** [comment]
Gjort

**Comment by danmichaelo** [comment]
Hver gang jeg ser "Klassisk elektrodynamikk" så stusser jeg litt. Tror jeg bytter om til "Klassisk elektromagnetisme" som foretrukket term.

**Comment by mittinatten** [comment]
Jeg tror jeg må si meg uenig :) Kanskje fordi jeg lærte meg faget gjennom denne https://en.wikipedia.org/wiki/Classical_Electrodynamics_(book)

**Comment by danmichaelo** [comment]
Men du ser at det står "Subject: Electromagnetism" i infoboksen på den Wikipedia-artikkelen ;)

Uansett, setter ombyttingen på pause da siden dette er så subjektivt. Jeg tror kanskje en grunn til at elektrodynamikk som hovedterm skurrer litt for meg er at introbøker typisk begynner med (elektro|magneto)statikk. Ville du lagt på egne emneord for dette, eller ville du tenkt at det går like greit under elektrodynamikk som elektromagnetisme?
